build/cluster.js

```diff
@@ -0,0 +1,92 @@
+// Copyright 2015 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Gulp tasks for kubernetes cluster management.
+ */
+import gulp from 'gulp';
+import chmod from 'gulp-chmod';
+import gunzip from 'gulp-gunzip';
+import untar from 'gulp-untar';
+import request from 'request';
+import source from 'vinyl-source-stream';
+import conf from './conf';
```

Comment:
There should be one line break between global imports and relative imports (i.e., `import './foo'`)